db/migrate/20181114213758_create_backup_code_configurations.rb

```diff
@@ -0,0 +1,14 @@
+class CreateRecoveryCodeConfigurations < ActiveRecord::Migration[5.1]
```

Comment:
This migration gives a 'NameError: uninitialized constant CreateBackupCodeConfigurations'.  The name of the file does not match the class name.

Reply:
fixed